jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -360,6 +361,12 @@ const DeploymentFlow = React.createClass({
       'Button click',
       'Deploy model'
     );
+    if (this.props.stats) {
+      const flags = window.juju_config.flags;
+      const statName = 'deploy' + (flags.ddeploy ? '.direct' : '.target');
```

Comment:
ddeploy is a flag set by the store front, maybe I should add a comment. We need to count deployments started with and without that flag.